From 52799e1dd66194f9fda205a946777f520f367f68 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ingo=20Kl=C3=B6cker?= <dev@ingo-kloecker.de>
Date: Wed, 14 May 2025 17:29:18 +0200
Subject: [PATCH] Include QGpgME/Debug for QDebug operator for GpgME::Error

The missing QDebug operator was caught by the new explicit
conversion-to-bool operator for GpgME::Error. The safe-bool idiom didn't
prevent conversion of GpgME::Error to bool when passed to QDebug's
operator<<.
---
 src/dialogs/useridswidget.cpp | 1 +
 src/view/cardkeysview.cpp     | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/dialogs/useridswidget.cpp b/src/dialogs/useridswidget.cpp
index bcb540b..fc26e7b 100644
--- a/src/dialogs/useridswidget.cpp
+++ b/src/dialogs/useridswidget.cpp
@@ -33,6 +33,7 @@
 #include <QPushButton>
 #include <QVBoxLayout>
 
+#include <QGpgME/Debug>
 #include <QGpgME/KeyListJob>
 #include <QGpgME/Protocol>
 
diff --git a/src/view/cardkeysview.cpp b/src/view/cardkeysview.cpp
index 3e7e6dd..305b8e2 100644
--- a/src/view/cardkeysview.cpp
+++ b/src/view/cardkeysview.cpp
@@ -35,6 +35,7 @@
 #include <KLocalizedString>
 #include <KSharedConfig>
 
+#include <QGpgME/Debug>
 #include <QGpgME/KeyListJob>
 #include <QGpgME/Protocol>
 
-- 
2.50.0

